R/DataJoint.R

```diff
@@ -117,7 +117,25 @@ setValidity(
 
 # DataJoint-as.list ----
 
-#' @rdname as_stan_list
+
+
+
+#' Data Object -> `list`
+#'
+#' @param object (`DataSubject` or `DataLongitudinal` or `DataSurvival`) \cr
+#' data object to convert to a `list`
+#' @param x (`DataSubject` or `DataLongitudinal` or `DataSurvival`) \cr
+#' data object to convert to a `list`
```

Comment:
```suggestion
#' @param object (`DataSubject` or `DataLongitudinal` or `DataSurvival`) \cr
#' data object to convert to a `list`.
#' @param x (`DataSubject` or `DataLongitudinal` or `DataSurvival`) \cr
#' data object to convert to a `list`.
```
(just for consistency)